integration_tests/mockApis/keyworkerApi.ts

```diff
@@ -254,6 +252,8 @@ export default {
   stubKeyworkerApiStatusFail: () => stubKeyworkerApiStatusFail(),
   stubKeyworkerApiStats2025,
   stubKeyworkerApiStats2024,
-  stubKeyworkerMigrationStatus,
+  stubKeyworkerMigrationStatus: () => stubKeyworkerPrisonConfig(true, true),
   stubKeyworkerApiStatsNoData,
+  stubPrisonNoHighRisk: () => stubKeyworkerPrisonConfig(true, false),
```

Comment:
Similarly best named stubEnabledPrisonNoHighComplex. They have high complexity of needs rather than being high risk prisoners